### PR TITLE
PHR-704 | Neeraj/Sangita | Adding new OtpRequest for Hip  Service

### PR DESCRIPTION
**Pull request by Nexengineer**

### CODE REVIEW

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Link/OtpCreationDetail.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5897932&bid=19311415)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Link/OtpCreationDetail.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5897932&bid=19311415)

File: src/In.ProjectEKA.HipService/Link/OtpCreationDetail.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+namespace In.ProjectEKA.HipService.Link
+{
+    public class OtpCreationDetail
+    {
+        public string SystemName { get; set; }
+        public string Action { get; set; }
+
+        public OtpCreationDetail(string systemName, string action)
```
Comment (leelanakka) [inline src/In.ProjectEKA.HipService/Link/OtpCreationDetail.cs:8]: Same as *OtpGenerationDetail*

Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/Link/OtpCreationDetail.cs]: done

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Link/OtpGenerationDetail.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5897932&bid=19311415)

**Pull request merged**